python/aixexplainer.Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM python:3.7
+FROM python:3.7-slim
```

Comment:
ideally we'd want to use `python:3.8-slim` which:
a. includes 5 critical CVEs (according to `grype`)
b. includes `gcc` and so there's no need for apt get and apt remove in this file.

however, `python:3.8` supports tensorflow >= 2.2 whilst for `aix360` versions 0.2.0 and 0.2.1 [demand tensorflow 1.14](https://github.com/Trusted-AI/AIX360/blob/1d29fe2046bfc2e738a7ba86977b86e8cedbaed8/setup.py#L26).
 in fact the extra critical CVEs reported in python:3.7-slim all come from tensorflow.
<img width="809" alt="image" src="https://user-images.githubusercontent.com/35975404/182411483-bfc6547f-af45-4763-bf56-f86110a23436.png">

For now, there's no other option unless aix360 update their dependency

Reply:
Let me open an issue to the aix360 community to update the tensorflow version.

Reply:
thank you Tommy! 
dependabot seems to have already [raised a PR](https://github.com/Trusted-AI/AIX360/pull/150) for that.